docs/src/main/sphinx/release/release-356.md

```diff
@@ -0,0 +1,73 @@
+# Release 356 (XX Apr 2021)
+
+## General
+
+* Add support for {doc}`/sql/match-recognize`. ({issue}`6111`)
+* Add {func}`soundex` function. ({issue}`4022`)  
+* Introduce `system.metadata.materialized_view_properties` table for listing available materialized view properties. ({issue}`7615`)
+* Introduce planning time limit which can be specified using `query.max-planning-time` config property ({issue}`7213`)
+* Improve cancellation of queries during planning phase ({issue}`7213`)
+* Improve performance of `ORDER BY ... LIMIT` queries over a `LEFT JOIN`. ({issue}`7028`)
+* Improve performance of queries with predicates on boolean columns ({issue}`7263`)
+* Improve planning time for queries with large `IN` predicates. ({issue}`7556`)
+* Improve performance of queries that contain join on `VARCHAR` keys of different length. ({issue}`7644`)
+* Reduce coordinator network overhead when scheduling queries. ({issue}`7351`)
+* Fix join spill deadlock. ({issue}`7455`)
+* Fix invalid query result for queries with full join which has scalar (left or right) input. ({issue}`7629`)
+* Add dedicated access control for creating and dropping materialized views. ({issue}`7645`)
+
+## JDBC driver
+
+* Add `externalAuthenticationTokenCache=MEMORY` property so Oauth2 credentials are cached in memory. Thanks to this user can avoid unnecessary authentication flow. ({issue}`7309`)
+
+## BigQuery connector
+
+* Add support for `CREATE SCHEMA` and `DROP SCHEMA` statements. ({issue}`7543`)
+* Improve performance for listing tables when case insensitive matching (`bigquery.case-insensitive-name-matching`) is enabled. ({issue}`7628`)
+
+## Cassandra connector
+
+* Fix `NullPointerException` when reading an empty timestamp value. ({issue}`7433`)
+
+## Hive connector
+
+* Fix incorrect results when non-lowercase field dereference is pushed into the ORC reader,
+  if ORC file schema has nested fields with a different case. ({issue}`7350`)
```

Comment:
```suggestion
* Fix incorrect results when non-lowercase field dereference is pushed into the ORC reader,
  if ORC file schema has nested fields with non-lowercase names. ({issue}`7350`)
```

IMO `different case` is not straightforward - different from what? The table definition (which is always lowercase IIRC)?